src/updater.h

```diff
@@ -35,16 +35,32 @@ class Configuration;
 class MetadataUpdater {
  public:
   struct ResourceMetadata {
-    ResourceMetadata(const std::vector<std::string>& ids_,
-                     const MonitoredResource& resource_,
-                     MetadataStore::Metadata&& metadata_)
-        : ids(ids_), resource(resource_), metadata(std::move(metadata_)) {}
+    ResourceMetadata(const std::vector<std::string>& ids,
+                     const MonitoredResource& resource,
+                     MetadataStore::Metadata&& metadata)
+        : ids_(ids), resource_(resource), metadata_(std::move(metadata)) {}
     ResourceMetadata(ResourceMetadata&& other)
-        : ResourceMetadata(other.ids, other.resource,
-                           std::move(other.metadata)) {}
-    std::vector<std::string> ids;
-    MonitoredResource resource;
-    MetadataStore::Metadata metadata;
+        : ResourceMetadata(other.ids_, other.resource_,
+                           std::move(other.metadata_)) {}
+
+    const MetadataStore::Metadata& metadata() const {
+      return metadata_;
```

Comment:
You should be able to put the whole function on one line, saving some vertical space. These are accessors, they don't deserve it. 😄 No need to put blank lines between them either.

Reply:
Done. lol.